src/sagemaker/estimator.py

```diff
@@ -601,6 +644,7 @@ def __init__(self, entry_point, source_dir=None, hyperparameters=None, enable_cl
                 Valid values are defined in the Python logging module.
             code_location (str): Name of the S3 bucket where custom code is uploaded (default: None).
                 If not specified, default bucket created by ``sagemaker.session.Session`` is used.
+            **kwargs: Additional kwargs passed to the ``EstimatorBase`` constructor.
```

Comment:
remove this line (it's already on l. 652)

Reply:
Now I'm curious how I even got that line there. I'll do a second pass over the code changes.